Enumerable.rb

```diff
@@ -0,0 +1,118 @@
+module Enumerable
+    def my_each
+      for i in 0..self.length-1
+        yield(self[i])
+      end
+    end
+    
+    def my_each_with_index
+        for i in 0..self.length-1
+            yield(self[i], i)
+        end
+    end
+
+    def my_select
+        array = []
+        self.my_each do |x|
+            if yield(x)
+                array << x
+            end
+        end
+        array
+    end
+    def my_all?
+        bool = true
+        self.my_each do |x|
+            if !yield(x)
+                bool = false
+            end
+        end
+        bool
+    end
+    def my_any?
+        bool = false
+        self.my_each do |x|
+            if yield(x)
+                bool = true
+            end
+        end
+        bool
+    end
+    def my_none?
```

Comment:
#### `REQUIRED`

#### '#my_none?` should be able to take an argument.

> If instead a pattern is supplied, the method returns whether pattern === element for none of the collection members.

##### Learn more [here](https://ruby-doc.org/core-2.6.3/Enumerable.html#method-i-none-3F).

```ruby
$ irb
irb(main):001:0> require_relative './Enumerable'
=> true
irb(main):002:0> [1,2,3,4,5].none?(3)
=> false
irb(main):003:0> [1,2,3,4,5].none?(9)
=> true
irb(main):004:0> [1,2,3,4,5].my_none?(9)
Traceback (most recent call last):
        5: from /home/bola/.rbenv/versions/2.6.3/bin/irb:23:in `<main>'
        4: from /home/bola/.rbenv/versions/2.6.3/bin/irb:23:in `load'
        3: from /home/bola/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/irb-1.0.0/exe/irb:11:in `<top (required)>'
        2: from (irb):4
        1: from /media/bola/.../code-reviews/Advanced_Building_Blocks/Enumerable.rb:41:in `my_none?'
ArgumentError (wrong number of arguments (given 1, expected 0))
irb(main):005:0> 
```